### ISSUE TITLE
Feature: Add headers CORS for use with web app

### ISSUE THREAD

**Issue by aarnaud** [lxc/lxd#1705]
It's possible to add in configuration this headers:

```
Access-Control-Allow-Methods: "GET, POST, PUT, DELETE, OPTIONS"
Access-Control-Allow-Headers: "Content-Type"
```

Like `lxc config set core.https_allowed_origin "*"`:

```
lxc config set core.https_allowed_methods "GET, POST, PUT, DELETE, OPTIONS"
lxc config set core.https_allowed_headers "Content-Type"
```

I works on webui with angular, i will push this on github.

Thanks,

**Comment by stgraber** [comment]
Sounds like a reasonable addition, a pull request would be welcome.

**Comment by aarnaud** [comment]
merged #1706
